pkg/services/dashboardversion/dashverimpl/store.go

```diff
@@ -0,0 +1,40 @@
+package dashverimpl
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/models"
+	dashver "github.com/grafana/grafana/pkg/services/dashboardversion"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/services/sqlstore/db"
+)
+
+type store interface {
+	Get(ctx context.Context, query *dashver.GetDashboardVersionQuery) (*dashver.DashboardVersion, error)
+}
+
+type sqlStore struct {
+	db db.DB
+}
+
+func (s *sqlStore) Get(ctx context.Context, query *dashver.GetDashboardVersionQuery) (*dashver.DashboardVersion, error) {
+	var version dashver.DashboardVersion
+	err := s.db.WithDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		has, err := sess.Where("dashboard_version.dashboard_id=? AND dashboard_version.version=? AND dashboard.org_id=?", query.DashboardID, query.Version, query.OrgID).
+			Join("LEFT", "dashboard", `dashboard.id = dashboard_version.dashboard_id`).
+			Get(&version)
+
```

Comment:
Good for now, but just to think about - could we make an in-memory JOIN instead (using the dashboard service, once that is complete)? Ideally we shouldn't access any other tables but the ones owned by the dashver service itself.